test/test-handler-spec.js

```diff
@@ -11,6 +11,7 @@ import options from './fixtures/options';
 describe('Test Handler', function() {
   it('should wait for all children to exit before returning', function () {
     var opts = options.default;
+    this.timeout(5000);
```

Comment:
the default is 2 seconds, but on windows, starting cucumber-js is pretty slow and takes up to 3 or 4 seconds.